app/src/keymap.c

```diff
@@ -191,11 +208,14 @@ int zmk_keymap_sensor_triggered(u8_t sensor_number, struct device *sensor)
 
 int keymap_listener(const struct zmk_event_header *eh)
 {
-	if (is_position_state_changed(eh)) {
+	if (is_position_state_changed(eh))
+	{
 		const struct position_state_changed *ev = cast_position_state_changed(eh);
 		return zmk_keymap_position_state_changed(ev->position, ev->state);
 #if ZMK_KEYMAP_HAS_SENSORS
-	} else if (is_sensor_event(eh)) {
+	}
```

Comment:
I am strongly against this for conditionals, and mixed feelings about this for function definitions. Please revert these style related changes until we can actually fully settle on something. Thanks!

Reply:
Reverting these changes will also make it easier to review just the fixes found here.